Client/Configuration/Ecosia.xcconfig

```diff
@@ -4,6 +4,7 @@
 
 #include "Common.xcconfig"
 #include "Release.xcconfig"
+#include "Production.xcconfig"
```

Comment:
Just wondering if we also need a step like [this](https://github.com/ecosia/ios-browser/blob/c49dab3b0c95bf334286d233f17ba787dbe9d184/.circleci/config.yml#L27) then to make CI work? 

`
name: extract Staging XCConfig
  command: echo "$STAGING_XCCONFIG" | base64 --decode > Client/Configuration/Staging.xcconfig
`
As `Production.xcconfig` is not in version control 🤔